packages/push/examples/smallest/args.rs

```diff
@@ -0,0 +1,41 @@
+use clap::Parser;
+
+#[derive(clap::ValueEnum, Copy, Clone, Debug)]
+pub enum RunModel {
+    Serial,
+    Parallel,
+}
+
+#[derive(clap::ValueEnum, Copy, Clone, Debug)]
+pub enum TargetProblem {
+    DegreeThree,
+}
```

Comment:
We should probably remove this like we did in all the other examples

Reply:
I totally never looked at `args.rs`, did I? Sighs. Glad you were more attentive!